assets/js/builder/Views/VideoExplainer.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Sidebar Utilities View
+ *
+ * @since [version]
+ * @version [version]
+ */
+define( [], function() {
+	return Backbone.View.extend( {
+
+		/**
+		 * HTML element selector
+		 *
+		 * @type {string}
+		 */
+		el: '#llms-video-explainer',
+
+		/**
+		 * Wrapper Tag name
+		 *
+		 * @type {string}
+		 */
+		tagName: 'div',
+
+		/**
+		 * Events
+		 *
+		 * @type {Object}
+		 */
+		events: {
+			'click .llms-video-explainer-trigger': 'openPopup',
+			'click .llms-video-explainer-close': 'closePopup',
+			'click .llms-video-explainer-wrapper': 'closePopup',
+		},
+
+		/**
+		 * Youtube video url
+		 *
+		 * @type {string}
+		 */
+		youtubeUrl: 'https://www.youtube.com/embed/kMd37cOsPIg',
```

Comment:
as I understand we use this URL in the `closePopUp` method, to re-set the `src` after cleaning it in order to stop the video. 
But we can actually get rid of this and retrieve the src directly from the iframe, no? (see suggestion in the relevant method).

Reply:
b6f05df